test/__mocks__/text.mock.ts

```diff
@@ -157,6 +157,7 @@ export const TRUST_NOT_ASSOCIATED_WITH_BENEFICIAL_OWNER_TEXT = "This trust is no
 export const TRUST_CEASED_DATE_TEXT = "When did this trust stop being associated with the overseas entity";
 export const TRUST_SELECT_TRUSTEES_TEXT = "Which of the overseas entity’s beneficial owners are trustees of this trust";
 export const TRUST_ENTER_CEASED_DATE = "Enter the date the trust stopped being associated with this overseas entity";
+export const TRUSTEE_STILL_INVOLVED_TEXT = "Are they still involved in the trust";
```

Comment:
This same question text is displayed for legal entities and individuals. As you're needing to add it now for the assertion in the test below, I wonder if we're missing a unit test (or just assertion) for the individual trustee page?

Reply:
i'll have a look

Reply:
updated